src/sagas/wallet.js

```diff
@@ -398,6 +405,14 @@ export function* listenForFeatureFlags(featureFlags) {
 
       if (oldUseWalletService && oldUseWalletService !== newUseWalletService) {
         yield put(reloadWalletRequested());
+        return;
+      }
+
+      const newShowAtomicSwap = yield take(channel);
+      const oldShowAtomicSwap = yield select((state) => state.useAtomicSwap);
+
+      if (oldShowAtomicSwap && oldShowAtomicSwap !== newShowAtomicSwap) {
+        yield put(reloadWalletRequested());
```

Comment:
Why do you do a full reload in the wallet if we change the atomic swap? I don't thinks this should be like this, in my opinion we don't need to update the atomic swap feature flag in real time like we do with the wallet service.

Reply:
I agree, it's enough to only check this flag on wallet startup.
Removed this realtime checking on [9fd4070](https://github.com/HathorNetwork/hathor-wallet/pull/361/commits/9fd4070a080e8d0a2d75156cb3b989ea3dfc2c41)